CameraBrowser/CameraBrowserSample/ViewController.cs

```diff
@@ -0,0 +1,238 @@
+﻿using System;
+using AppKit;
+using Foundation;
+using ImageCaptureCore;
+using ObjCRuntime;
+#nullable enable
+
+namespace CameraBrowserSample
+{
+	public partial class ViewController : NSViewController, IICDeviceBrowserDelegate, IICCameraDeviceDelegate, IICCameraDeviceDownloadDelegate, IICDeviceDelegate {
+		public ICDeviceBrowser DeviceBrowser { get; set; } = new ICDeviceBrowser();
+		public NSMutableArray Cameras { [Export("Cameras")] get; [Export("setCameras:")] set; } = new NSMutableArray();
+
+		static ViewController() //init
+		{
+			var imageTransformer = new CGImageRefToNSImageTransformer();
+			NSValueTransformer.SetValueTransformer(imageTransformer, "CGImageRefToNSImage");
+		}
+
+		public ViewController (IntPtr handle) : base(handle)
+		{
+		}
+
+		public override void ViewWillAppear()
+		{
+			base.ViewWillAppear();
+			View.Window.Title = "Camera Browser Sample for Xamarin.Mac ";
+		}
+
+		public void HandleProcessICLaunchParams(NSNotification note) => Console.WriteLine($"NSNotification: {note}");
+
+		public override void ViewDidLoad () //application did finish launching
+		{
+			base.ViewDidLoad ();
+
+			// Do any additional setup after loading the view.
+
+			CamerasController.SelectsInsertedObjects = false;
+
+			NSNotificationCenter.DefaultCenter.AddObserver(this, new Selector("processICLaunchParams:"), (NSString)"ICLaunchParamsNotification", null);
+
+			CamerasController.SelectsInsertedObjects = false;
+
+			MediaFilesController.AddObserver(this, (NSString)"selectedObjects", (NSKeyValueObservingOptions)0, IntPtr.Zero);
```

Comment:
Same:

```suggestion
			MediaFilesController.AddObserver("selectedObjects", (NSKeyValueObservingOptions)0, (v) => {
				WillChangeValue("canDelete");
				WillChangeValue("canDownload");
				DidChangeValue("canDelete");
				DidChangeValue("canDownload");
			});
```

Also note that "(NSKeyValueObservingOptions) 0" looked clunky, so I'm adding a None option: https://github.com/xamarin/xamarin-macios/pull/17365, so that you can do "NSKeyValueObservingOptions.None" one day.